src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6676,6 +6676,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UnreadRecordParameter_Title" xml:space="preserve">
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
+  <data name="IDS_FeatureReducedMemberAccessChecksInNameof" xml:space="preserve">
+    <value>reduced member access checks in 'nameof'</value>
```

Comment:
>reduced member access checks in 'nameof' [](http://example.com/codeflow?start=11&length=40)

Is it possible to come up with a more specific feature name? Let's say to morrow we will implement more relaxations, how would we call them then?

Reply:
Before I go ahead and perform replace all, can you confirm the following would work? (Note that I changed both the name and value.)

```xml
  <data name="IDS_FeatureNestedInstanceMemberAccessInStaticNameof" xml:space="preserve">
    <value>allow nested instance member access in 'nameof' in static context</value>
  </data>
```

Reply:
> Before I go ahead and perform replace all, can you confirm the following would work?

Feels too lengthy to me. I would drop the "nested" and "access", at least. Then we will be left with: "allow instance member in 'nameof' in static context." Could we shorten it to just "instance member in 'nameof'"?

@333fred What do you think?

Reply:
I think "instance member in 'nameof'" is sufficiently descriptive.